From 57894a6d01c91ab84bbf41616034cc534b0c4537 Mon Sep 17 00:00:00 2001
From: Mingke Wang <mingke.wang@freescale.com>
Date: Thu, 19 Mar 2015 14:15:25 +0800
Subject: [PATCH 02/91] gstplaysink: don't set async of custom text-sink to
 false

set async to false lead to A/V sync problem when seeking.
the preroll need use GAP event instead of set async to false.

Upstream-Status: Inappropriate [i.MX specific]

Signed-off-by: Mingke Wang <mingke.wang@freescale.com>
(cherry picked from commit 8ec4cab5637f44baaf4a2c39bb308db29c9793d9)
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 gst/playback/gstplaysink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/playback/gstplaysink.c b/gst/playback/gstplaysink.c
index f4d7a0f..f2dc616 100644
--- a/gst/playback/gstplaysink.c
+++ b/gst/playback/gstplaysink.c
@@ -2485,7 +2485,7 @@ gen_text_chain (GstPlaySink * playsink)
           G_TYPE_BOOLEAN);
       if (elem) {
         /* make sure the sparse subtitles don't participate in the preroll */
-        g_object_set (elem, "async", FALSE, NULL);
+        //g_object_set (elem, "async", FALSE, NULL);
         GST_DEBUG_OBJECT (playsink, "adding custom text sink");
         gst_bin_add (bin, chain->sink);
         /* NOTE streamsynchronizer needs streams decoupled */
-- 
2.43.0

